src/components/aside/Aside.js

```diff
@@ -1,7 +1,21 @@
 import './aside.scss';
 
+const RenderNums = ({
+  from, to, odd, even
+}) => {
+  const items = [];
+
+  for (let i = +from; i <= to; i++) {
+    odd && i % 2 === 1 && items.push(<li key={i}>{i}</li>);
+    even && i % 2 === 0 && items.push(<li key={i}>{i}</li>);
+    !odd && !even && items.push(<li key={i}>{i}</li>);
```

Comment:
I see in `push()` the same expression three times `<li key={i}>{i}</li>`. So, difference is only in `i` variable.
Please, refactor this block of code using `<li key={i}>{i}</li>` only one time